src/lib/djerba/configure.py

```diff
@@ -158,6 +177,19 @@ def run(self, out_path):
             self.config.write(out_file)
         self.logger.info("Djerba config finished; wrote output to {0}".format(out_path))
 
+    def try_coverage(self,coverage,target):
+        if target > coverage:
+            msg = "Target Depth {0}X is larger than Discovered Coverage {1}X. Changing to Failed mode.".format(target, coverage)
+            self.logger.warning(msg)
+            self.failed = True
+        elif target <= coverage:
+            msg = "Target Depth {0}X is within range of Discovered Coverage {1}X".format(target, coverage)
+            self.logger.info(msg)
+        else:
```

Comment:
If we get this far, something bad is happening and we should throw an error. Eg. we have somehow made either target or coverage a `NaN` value:

>>> import math
>>> foo = math.nan
>>> foo < 42
False
>>> foo >= 42
False
>>> None < 42
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: '<' not supported between instances of 'NoneType' and 'int'
>>> 'foobar' < 42
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: '<' not supported between instances of 'str' and 'int'

Reply:
do you agree switching this logger function from an 'info' call to an 'error' call would address your concerns?